### PR TITLE
Update my account page

### PR DESCRIPTION
**Pull request by hubertpompecki**
Related PR: https://github.com/shiftcommerce/shift-front-end-react/issues/837

Updates the my account page to render the parent `Sidebar` component.

### CODE REVIEW

File: test/pages/__snapshots__/cart.test.js.snap [hunk 1]
```diff
@@ -1156,11 +1156,11 @@ exports[`dispatch updateQuantity action on changing line item quantity 1`] = `
                   </link_Link>
                   <link_Link
                     className="o-button--sml c-cart-summary-buttons__cta c-cart-summary-buttons__cta--proceed"
-                    href="/checkout/shipping-address"
+                    href="/checkout/payment-method"
```
Comment (hubertpompecki) [inline test/pages/__snapshots__/cart.test.js.snap]: I believe changes here and in the `shipping-method` snapshot are due to the changes in shift-react-components 0.5.0 which was not pulled into shift next yet.

File: test/pages/my-account.test.js [hunk 2]
```diff
@@ -1,69 +1,74 @@
...
     wrapper.instance().componentDidMount()
 
-    expect(dispatch).toHaveBeenCalled()
+    expect(replaceSpy).toHaveBeenCalledWith('example.com?menu=Details')
+    expect(wrapper.state().currentMenu).toEqual('Details')
+
+    replaceSpy.mockRestore()
+  })
+
+  test("sets the state and doesn't redirect when a valid menu is given", () => {
```
Comment (Azfletch) [inline test/pages/my-account.test.js]: using a mix of double and single quotes in this file

Comment (hubertpompecki) [reply test/pages/my-account.test.js]: This is because there is an apostrophe in the string - it is an exception that eslint allows for.

Comment (Azfletch) [reply test/pages/my-account.test.js]: Didn't see that. No worries!

**Pull request merged**